urdf/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ add_compile_options(-std=c++11)
 
 catkin_package(
   LIBRARIES ${PROJECT_NAME}
-  INCLUDE_DIRS include ${TinyXML_INLCLUDE_DIRS} ${CATKIN_DEVEL_PREFIX}/include
+  INCLUDE_DIRS include ${TinyXML2_INLCLUDE_DIRS} ${CATKIN_DEVEL_PREFIX}/include
```

Comment:
I would rather we change the APIs to not expose TinyXML2 so its headers don't need to be exported.